bc_obps/registration/migrations/0059_auto_20241125_2153.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 5.0.9 on 2024-11-25 21:53
+
+from django.db import migrations
```

Comment:
Late-breaking thought--this should probably be in common, not registration

Reply:
![image](https://github.com/user-attachments/assets/ff8cf365-c209-4683-9435-c81c54108fc0)